new/app/loginPage/index.tsx

```diff
@@ -33,8 +32,8 @@ export default function LoginPage() {
     return (
         <>
             <BasicForm onSubmitCallback={submitFunc}>
-                <BasicTextInput name="uid" label="User Id" type="username"/>
-                <BasicTextInput name="password" label="Password" type="password"/>
+                <BasicTextInput name="userId" label="User Id" type="username" />
+                <BasicTextInput name="logInPasswordHashed" label="Password" type="password" />
```

Comment:
why it is named logInPasswordHashed? think the value is not hashed yet?

Reply:
This is fixed